tests/server/server-executeBulkWrite-001.phpt

```diff
@@ -70,7 +70,9 @@ object(MongoDB\Driver\WriteResult)#%d (%d) {
   array(0) {
   }
   ["writeConcern"]=>
-  array(4) {
+  array(5) {
```

Comment:
This is something @bjori brought up before. Since I'm modifying this line, it'd be reasonable to make the switch to `%d`. Thanks for noticing.